### PR TITLE
Ch2 Shell

### PR DESCRIPTION
**Pull request by joelostblom**
This is a WIP but promoting from draft to get Netlify to build.

### CODE REVIEW

**Comment by joelostblom** [comment]
@flor14 Ready for review! One particular thing I wonder what you think of is whether there should be one brief explaining sentence for each code cell on the slide itself or if you think it is enough that the commands are explained in the transcript.

File: chapters/en/slides/module2/module2_05.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+code my-first-note.md
+```
+
+Notes:
+Since we just created the `notes` directory,
+ls doesn't display anything when we ask for a listing of its contents.
+
+Let’s change our working directory to `notes`,
+then use the VS Code editor to create a file called my-first-note.md.
+The command to invoke VS Code from the command line is `code`,
```
Comment (flor14) [inline chapters/en/slides/module2/module2_05.md:62]: Should be good to add something as... "You can try it now!" So they start interacting with the material

File: chapters/en/slides/module2/module2_05.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+You will see that there is a small white circle next to the file name.
+This is an indication that the file is not saved yet.
+
+When we are done adding our edits,
+we can hit `Ctrl` + `s` to save the file,
+which will make the white circle disappear.
+After saving the file,
+you can optionally close VS Code before returning to the terminal.
+
+We are only using VS Code as a simple text editor here,
```
Comment (flor14) [inline chapters/en/slides/module2/module2_05.md:92]:
```suggestion
We are only using `VS Code` as a simple text editor here,
```

File: chapters/en/slides/module2/module2_05.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+Notes:
+This is what it looks like to edit a text file in VS Code.
+You will see that there is a small white circle next to the file name.
+This is an indication that the file is not saved yet.
+
+When we are done adding our edits,
+we can hit `Ctrl` + `s` to save the file,
+which will make the white circle disappear.
+After saving the file,
+you can optionally close VS Code before returning to the terminal.
```
Comment (flor14) [inline chapters/en/slides/module2/module2_05.md:90]:
```suggestion
you can optionally close `VS Code` before returning to the terminal.
```

File: chapters/en/slides/module2/module2_end.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+type: slides
+---
+
+# What Did we Learn and What to Expect in Assignment 2
```
Comment (flor14) [inline chapters/en/slides/module2/module2_end.md:5]: Is this section going to be included in all the modules?

File: chapters/en/slides/module2/module2_07.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+but you don't remember exactly which flags you used.
+
+We just learned about `grep` for searching for text in files,
+so maybe there is someway we could use it
+to search the text output from the history command?
+In other words,
+could we redirect the output from the history command to `grep`
+before printing it on the screen?
+
+This is exactly what a `pipe` (`|`) does,
```
[the remark this answers is not in the document]
Comment (joelostblom) [reply chapters/en/slides/module2/module2_07.md:85]: Yeah, I didn't want to make it too complex but I will include one more advanced command just to show the power of pipes.

**Pull request merged**